File: signing-consentor-archetype/src/main/resources/archetype-resources/src/main/java/__pluginName__/__pluginName__ConsentorConfig.java
package ${package}.${pluginName};

import se.curity.identityserver.sdk.config.Configuration;
import se.curity.identityserver.sdk.config.annotation.Description;

public interface ${pluginName}ConsentorConfig extends Configuration {
    
    @Description("A parameter used for demo purposes")
    Integer getDemoParameter();
}

File: kotlin-consentor-archetype/src/main/resources/archetype-resources/src/main/kotlin/__pluginName__/__pluginName__ConsentorConfig.kt
package ${package}.${pluginName}

import se.curity.identityserver.sdk.config.Configuration
import se.curity.identityserver.sdk.config.annotation.Description

interface ${pluginName}ConsentorConfig: Configuration
{
    @Description("A parameter used for demo purposes")
    fun getDemoParameter(): Int
}
